Sources/Elementary/Core/Html+Attributes.swift

```diff
@@ -79,6 +79,15 @@ public extension HTMLElement {
         self.attributes = .init(attributes)
         self.content = content()
     }
+
+    /// Creates a new HTML element with the specified attributes and content.
+    /// - Parameters:
+    ///  - attributes: The attributes to apply to the element as an array.
+    ///  - content: The content of the element.
+    init(_ attributes: [HTMLAttribute<Tag>], @HTMLBuilder content: () -> Content) {
+        self.attributes = .init(attributes)
+        self.content = content()
+    }
 }
 
 public extension HTMLVoidElement {
```

Comment:
we should add the same new array-based initializer overload to `HTMLVoidElement`